src/tools/kata-ctl/src/ops/volume_ops.rs

```diff
@@ -158,10 +158,17 @@ mod tests {
     use super::*;
     use kata_types::mount::DirectVolumeMountInfo;
     use serial_test::serial;
+    use sha2::{Digest, Sha256};
     use std::{collections::HashMap, fs, path::PathBuf};
     use tempfile::tempdir;
     use test_utils::skip_if_not_root;
 
+    fn sha256_hex_digest(input: &str) -> String {
```

Comment:
IMO，this is a public method not only for `kata-ctl` also for `join_path`.

Reply:
Done.